controllers/machinedeployment_sync_test.go

```diff
@@ -397,3 +398,105 @@ func TestScaleMachineSet(t *testing.T) {
 		})
 	}
 }
+
+func newDeploymentStatus(replicas, updatedReplicas, availableReplicas int32) clusterv1.MachineDeploymentStatus {
+	return clusterv1.MachineDeploymentStatus{
+		Replicas:          replicas,
+		UpdatedReplicas:   updatedReplicas,
+		AvailableReplicas: availableReplicas,
+	}
+}
+
+func currentDeployment(pds *int32, replicas, statusReplicas, updatedReplicas, availableReplicas int32, conditions clusterv1.Conditions) *clusterv1.MachineDeployment {
```

Comment:
`newTestMachineDeployment` may be is more meaningful? the "current" temporary nuance make no sense within the function scope. It's up to the caller defining this into a variable to choose that nuance.

Reply:
done